apps/mobile-wallet/src/analytics.tsx

```diff
@@ -69,6 +69,8 @@ export const sendAnalytics = (params: AnalyticsParams) => {
   } else {
     const { event, props, options } = params
 
+    if (props) props.$ip = ''
+
```

Comment:
Inspired by: https://posthog.com/questions/override-event-ip-address

This shouldn't be necessary since we already do that a few lines above (line 35).